src/tdvpinfo.jl

```diff
@@ -1,3 +1,8 @@
+"""
+#fields
+- `maxtruncerr::Float64`: the maximum tuncation error
+- `converged::Int`: 1 if converged, 0 if not, -1 if NA
```

Comment:
We can remove this part of the docstring now.

Reply:
I've done this. And also ran the formatter. The next run of jobs should be succesful (famous last words).